src/App.js

```diff
@@ -117,6 +118,7 @@ function App() {
         </div>
 
       </div>
+      <div className='item'><ModalWindow /></div>
```

Comment:
don't name your classes as `item`, this is not informative. always think about good names for your variables/classes. in that cases your code will be much easier to read. it will basically read like a story